### ISSUE TITLE
unpkg bunlde - standalone package errors

### ISSUE THREAD

**Issue by madhums** [lukasbach/react-complex-tree#110]
I am trying to use the unpkg bundle as mentioned in the [documentation](https://rct.lukasbach.com/docs/getstarted#using-cdns) but I get an error when the library loads. 

Here's a simple html file loading the required libraries, please check the console for the error

```html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>RCT</title>
  <script src="https://unpkg.com/react@17.0.2/umd/react.development.js"></script>
  <script src="https://unpkg.com/react-dom@17.0.2/umd/react-dom.development.js"></script>
  <script src="https://unpkg.com/react-complex-tree/lib/bundle.js"></script>
</head>
<body>
  Check if rct works
</body>
</html>
```

Error in web console:

```
InteractionManagerProvider.tsx:29 Uncaught TypeError: a.createContext is not a function
    at Object.<anonymous> (InteractionManagerProvider.tsx:29:39)
    at n (bootstrap:19:22)
    at Object.<anonymous> (ControlledTreeEnvironment.tsx:36:36)
    at n (bootstrap:19:22)
    at Object.<anonymous> (index.ts:13:14)
    at n (bootstrap:19:22)
    at bootstrap:83:10
    at bundle.js:1:1204
    at universalModuleDefinition:9:30
    at universalModuleDefinition:1:1
```

Is this a build issue? Has this been tested?

**Comment by madhums** [comment]
Apparently the externals are declared to be `react` and `reactDOM`, while by default unpkg exposes as `React` and `ReactDOM`.

So adding a 

```html
<script>
    var react = React;
    var reactDOM = ReactDOM;
  </script>
```

after importing react and react-dom solves the issue.

Is there a better way to do this?

**Comment by lukasbach** [comment]
My bad, fixed in v1.1.10. Thanks for figuring out the culprit.

**Comment by madhums** [comment]
Thank you for the quick response and release! Appreciate it!
